plugins/BEdita/Core/src/Model/Entity/Tag.php

```diff
@@ -51,18 +46,5 @@ class Tag extends Entity implements JsonApiSerializable
      */
     protected $_hidden = [
         '_joinData',
```

Comment:
If I'm not wrong `object_tags` doesn't have join data. I think we can remove `$_hidden` property.

Reply:
Yeah, correct.

```suggestion
```